web-hosting-case-study.md

```diff
@@ -0,0 +1,121 @@
+---
+title: web hosting
+keywords: web hosting
+tags: web hosting
+sidebar: development-platforms
+permalink: development-platforms/example-product.html
+folder: development-platforms
+---
+
+## Introduction
+
+### Summarise the product's primary purpose in one or two paragraphs.
```

Comment:
You can remove these prompts - they are just there to help you structure your work
```suggestion
```